src/html/HTMLWriter.jl

```diff
@@ -713,6 +713,61 @@ function SearchRecord(ctx, navnode, node::Node, ::MarkdownAST.AbstractElement)
     return SearchRecord(ctx, navnode; text = mdflatten(node))
 end
 
+
+# for @example blocks
+function SearchRecord(ctx::HTMLContext, navnode::Documenter.NavNode, node::MarkdownAST.Node{Nothing}, element::Documenter.MultiOutput)
+    @info "MultiOutput SearchRecord method called!"
+    children_array = collect(node.children)
+
+    if isempty(children_array) || length(children_array) < 2
+        return SearchRecord(ctx, navnode; text = "")
+    end
+
+    output_text = ""
+    for child in children_array[2:end]
+        if isa(child.element, Documenter.MultiOutputElement) && isa(child.element.element, Dict)
+            for mime in [MIME"text/plain"(), MIME"text/markdown"(), MIME"text/html"()]
+                if haskey(child.element.element, mime)
+                    output_text *= string(child.element.element[mime]) * " "
+                    break
+                end
+            end
+        else 
+            output_text *= mdflatten(child) * " "
+        end
```

Comment:
So, for at-example blocks, we render both the input and the output. We probably want to put the source code into the output as well?

![image](https://github.com/user-attachments/assets/34997cd6-3192-4888-bd96-65b56ba4d0e5)

In general, though, I wonder if it's better to just add the search index entries in the actual `domify` methods, like here:

https://github.com/JuliaDocs/Documenter.jl/blob/4328aec711dd3ee04d1989c796b241f547962fcd/src/html/HTMLWriter.jl#L2394-L2527

X-ref https://github.com/JuliaDocs/Documenter.jl/pull/2672#issuecomment-2767917688 about how to remove the other entries.